wmn-data.json

```diff
@@ -1240,6 +1240,18 @@
         "cat" : "art",
         "valid" : true
        },        
+       {
+        "name" : "counter_social",
+        "uri_check" : "https://counter.social/@{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "@counter.social) - CounterSocial</title>",
+        "m_string" : "The page you are looking for isn't here.",

```

Comment:
Can you please verify the `m_code`? In Chrome I see the `'` is `&39;` so the `m_code` value should be `The page you are looking for isn&#39;t here.`